src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="block">
```

Comment:
please read requirements, it should be calendar, not just block
![image](https://github.com/mate-academy/layout_calendar/assets/102289654/ae0ee772-378d-4620-bb09-52901d6b00f9)